src/styles/store.scss

```diff
@@ -0,0 +1,108 @@
+.store {
+
+  &__header {
+    font-size: 32px;
+
+    @include onTablet {
+      font-size: 40px;
+      width: 85%;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+      width: 65%;
+    }
+  }
+
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+
+    @include onTablet {
+      row-gap: 56px;
+    }
+  }
+
+  &__images {
+    display: grid;
+    grid-template-columns: repeat(2, 130px);
+    column-gap: 20px;
+
+    margin-bottom: 16px;
+
+    @include onTablet {
+      grid-template-columns: repeat(6, 80px);
+      column-gap: 24px;
+    }
+
+    @include onDesktop {
+      grid-template-columns: repeat(12, 63px);
+    }
+  }
+
+  &__link {
+
+    &--wide {
+      @include onTablet {
+        grid-column: span 4;
+      }
+      @include onDesktop {
+        grid-column: span 8;
+      }
+    }
+
+    &--square {
+      @include onTablet {
+        grid-column: span 2;
+      }
+      @include onDesktop {
+        grid-column: span 4;
+      }
+    }
+  }
+
+  &__image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+
+    @include onDesktop {
+      @include hover(transform, scale(1.05));
+    }
+  }
+
+  &__title {
+    font-size: 18px;
+
+    @include onDesktop {
+      font-size: 24px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    background-color: $main-color;
+    height: 56px;
+    align-items: center;
+    border-radius: 8px;
+    color: $btn-text-color;
+    margin-bottom: 96px;
+
+    &:active {
+      background-color: $main-color;
+      color: $btn-text-color;
+    }
+
+    @include onDesktop {
+      width: 324px;
+      transition: all 0.3s;
```

Comment:
It is better to set the `transition` for a specific parameter, and not for `all` at once.